src/cc65/compile.c

```diff
@@ -300,7 +299,7 @@ void Compile (const char* FileName)
     struct tm*  TM;
 
     /* Since strftime is locale dependent, we need the abbreviated month names
-    ** in english.
+    ** in English.
```

Comment:
The name of a country and the name of the people who live there are "capitated" (have an initial capital letter).  But, the name of their language isn't capitated.  Example:

**The French (who live in France) speak french.**